### PR TITLE
Move to es6 and sass

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

**Comment by matzimowski** [comment]
es6 ❤️❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ 
👍

**Comment by vergilius** [comment]
❤️ ❤️

**Comment by vergilius** [comment]
🐭

File: src/js/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+class SimpleFlipClock {
+  constructor(element, currentNumber, maxNumber = 9, minNumber = 0) {
```
Comment (kdzwinel) [inline src/js/main.js]: `{element, currentNumber, maxNumber = 9, minNumber = 0}`

**Comment by kdzwinel** [comment]
ok

**Comment by kdzwinel** [comment]
dist js file is `.min.js` but is not minified?

**Comment by aju** [comment]
@kdzwinel its minified - please verify

**Comment by aju** [comment]
@kdzwinel construtor function updated

**Comment by kdzwinel** [comment]
ok!

**Pull request merged**